easybuild/easyconfigs/g/GCCcore/GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '11.3.0'
+
+homepage = 'https://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+    'https://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
+    'https://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
+    'https://ftpmirror.gnu.org/gnu/mpc',  # idem for MPC
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+    'https://sourceware.org/pub/newlib/',  # for newlib
+    'https://github.com/MentorEmbedded/nvptx-tools/archive',  # for nvptx-tools
+]
+sources = [
+    'gcc-%(version)s.tar.gz',
+    'gmp-6.2.1.tar.bz2',
+    'mpfr-4.1.0.tar.bz2',
+    'mpc-1.2.1.tar.gz',
+    'isl-0.24.tar.bz2',
+    'newlib-4.1.0.tar.gz',
+    {'download_filename': 'd0524fb.tar.gz', 'filename': 'nvptx-tools-20210115.tar.gz'},
```

Comment:
@SebastianAchilles This should be updated to latest? (see https://github.com/MentorEmbedded/nvptx-tools/)

```python
{'download_filename': '7292758.tar.gz', 'filename': 'nvptx-tools-20220412.tar.gz'},
```

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15311/commits/a2d5095030c2cc58d9111e85395ba90a64fcc377